static/home/css/style.css

```diff
@@ -702,35 +699,34 @@ a:hover{
 	justify-self: center;
 }
 
-.contact-details-div {
+.don-box3 .contact-details-div {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
 	margin-bottom: 12px;
 	
 }
 
-.details-1 {
+.details-1, .details-2 {
 	text-align: center;
     max-width: 150px;
     margin-bottom: 12px;
 	margin-top: 12px;
+	
+}
+.details-1 {
 	margin-left: 64px;
 }
 
 .details-2 {
-	text-align: center;
-    max-width: 150px;
-    margin-bottom: 12px;
-	margin-top: 12px;
 	margin-right: 64px;
 }
 
-.don-box3-p {
+.or {
```

Comment:
You should write this class as `.don-box3 .or`, as `.or` class can appear at other places too, so you must be specific that these properties should only apply to the `.or` class inside `.don-box3` class and not anywhere else.